server/stream.go

```diff
@@ -1069,10 +1160,10 @@ func (mset *stream) setupMirrorConsumer() error {
 			DeliverSubject: string(sub.subject),
 			DeliverPolicy:  DeliverByStartSequence,
 			OptStartSeq:    state.LastSeq,
-			AckPolicy:      AckExplicit,
+			AckPolicy:      AckNone,
 			AckWait:        48 * time.Hour,
 			MaxDeliver:     1,
-			MaxAckPending:  mirrorMaxAckPending,
+			RateLimit:      1 * 1024 * 1024 * 1024, // 1 Gbit
```

Comment:
wont this increase likelyhood of triggering slow consumers on links incapable of handling that?

Reply:
I lowered it and still thinking through better ways we might approach this.